GDJS/Runtime/events-tools/stringtools.js

```diff
@@ -88,33 +88,51 @@ gdjs.evtTools.string.strLen = function(str) {
 };
 
 /**
- * Search in a string
+ * Search the first occurence in a string (return the position of the result, from the beginning of the string, or -1 if not found)
  * @private
  */
 gdjs.evtTools.string.strFind = function(str, what) {
     return str.indexOf(what);
 };
 
 /**
- * Reverse search in a string
+ * Search the last occurence in a string (return the position of the result, from the beginning of the string, or -1 if not found)
  * @private
+ * @deprecated
  */
 gdjs.evtTools.string.strRFind = function(str, what) {
     return str.lastIndexOf(what);
```

Comment:
As it's deprecated, let's implement it in terms of the not deprecated method:
`return gdjs.evtTools.string.strFindLast(str, what);`

Reply:
Actually, `gdjs.evtTools.string.strRFind = gdjs.evtTools.string.strFindLast;` will do the trick ;)